src/controllers/update/update.any.trusts.involved.controller.ts

```diff
@@ -0,0 +1,30 @@
+import { NextFunction, Request, Response } from "express";
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+import { AnyTrustsInvolvedKey, yesNoResponse } from "../../model/data.types.model";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+    return res.render(config.UPDATE_ANY_TRUSTS_INVOLVED_PAGE, {
+      backLinkUrl: config.SECURE_UPDATE_FILTER_URL,
+      templateName: config.UPDATE_ANY_TRUSTS_INVOLVED_PAGE
+    });
+  } catch (error) {
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+    const anyTrustsInvolved = (req.body[AnyTrustsInvolvedKey]) ? +req.body[AnyTrustsInvolvedKey] : 0;
```

Comment:
I think this has two different types and maybe check for contains rather than true would be better? 
So could be:
const anyTrustsInvolved = AnyTrustsInvolvedKey in req.body ? +req.body[AnyTrustsInvolvedKey] : "0";

Reply:
The validation should really make sure the selection was made so really all we need is:
if (req.body[AnyTrustsInvolvedKey] === yesNoResponse.Yes) {